ShowPot/ShowPot/Presentation/Common/BaseViewController.swift

```diff
@@ -7,8 +7,47 @@
 
 import Foundation
 import UIKit
```

Comment:
큰 부분은 아니지만 UIKit안에 Foundation이 존재하니 Foundation은 삭제해도 좋을꺼같아요 !!
```suggestion
import UIKit
```